examples/multimodal/vision_language_foundation/clip/convert_external_clip_to_nemo.py

```diff
@@ -55,8 +55,8 @@
 
 def get_args():
     parser = ArgumentParser()
-    parser.add_argument("--arch", type=str, default="ViT-H-14")
-    parser.add_argument("--version", type=str, default="laion2b_s32b_b79k")
+    parser.add_argument("--arch", type=str, default="huggingface")
+    parser.add_argument("--version", type=str, default="openai/clip-vit-base-patch32")
```

Comment:
note for hugging face, you should set version to huggingface and set arch to model name, check line 255 below

Reply:
Thanks, @yaoyu-33, for the comment. I resolved the typo.

Reply:
Attached a picture that shows the successful run of [convert_external_clip_to_nemo.py]